pkg/machine/qemu/machine.go

```diff
@@ -1411,3 +1420,53 @@ func (v *MachineVM) getImageFile() string {
 func (v *MachineVM) getIgnitionFile() string {
 	return v.IgnitionFilePath.GetPath()
 }
+
+func (v *MachineVM) resizeDisk(diskSize uint64) error {
+	originalDiskSize, err := getDiskSize(v.getImageFile())
+	if err != nil {
+		return err
+	}
+	// Resize the disk image to input disk size
+	// only if the virtualdisk size is less than
+	// the given disk size
+	if diskSize<<(10*3) <= originalDiskSize {
```

Comment:
Can you add a comment what this <<30 should do. I assume it converts bytes to mb but a comment should help.